src/syscall/os_wasip1.go

```diff
@@ -5,5 +5,9 @@
 package syscall
 
 func ProcExit(code int32) {
-	return
+	if code == 0 {
+		wasmPause()
```

Comment:
I am not sure the file will be included in the compilation. The reason is that wasmPause() and require(0) are private functions in runtime package, which cannot be accessed in syscall package.